### PR TITLE
frontend: inject db handle into schemaResolver

### PR DESCRIPTION
**Pull request by asdine**
This gets the global dbconn and pass it as a member to the graphql SchemaResolver.
The handle will be used by resolvers to create stores and call them without using dbconn.Global.

### CODE REVIEW

File: cmd/frontend/graphqlbackend/testing.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ func mustParseGraphQLSchema(t *testing.T) *graphql.Schema {
 	t.Helper()
 
 	parseSchemaOnce.Do(func() {
-		parsedSchema, parseSchemaErr = NewSchema(nil, nil, nil, nil, nil, nil)
+		parsedSchema, parseSchemaErr = NewSchema(nil, nil, nil, nil, nil, nil, nil)
```
Comment (unknwon) [inline cmd/frontend/graphqlbackend/testing.go:20]: This makes me think we should start using a struct 😂 (not related to this PR)

File: cmd/frontend/internal/cli/serve_cmd.go [hunk 2]
```diff
@@ -100,15 +101,15 @@ func InitDB() error {
 
 		err := backend.UpdateServiceVersion(ctx, "frontend", version.Version())
 		if err != nil && !dbutil.IsPostgresError(err, "42P01") {
-			return err
+			return nil, err
 		}
 
 		if !migrate {
-			return nil
+			return dbconn.Global, nil
```
Comment (unknwon) [inline cmd/frontend/internal/cli/serve_cmd.go:108]: What's the reason that we need to return `dbcoon.Global` here?

**Review by unknwon — APPROVED** [review]

File: cmd/frontend/graphqlbackend/testing.go (hunk 1 above)
Comment (asdine) [reply cmd/frontend/graphqlbackend/testing.go:20]: Definitely!

File: cmd/frontend/internal/cli/serve_cmd.go (hunk 2 above)
Comment (asdine) [reply cmd/frontend/internal/cli/serve_cmd.go:108]: This is a short step in moving away from dbconn.Global: we are going to use it both globally and locally (using the schemaResolver db struct member) until all the calls to our stores use the local version.

**Pull request merged**